### PR TITLE
Article Style Guide updates

### PR DESCRIPTION
**Pull request by MegaApplePi**
- added which point of view to use
- fixed things in the Miscellaneous section
- noted to use `Multi`, not `Multiplayer`
- noted types of links (absolute, relative-absolute, and relative-relative)
- fixed things in the Code section
- fixed things in the Links section
- fixed things in the Miscellaneous section
- ~~**noted to use `new.ppy.sh` instead of `osu.ppy.sh`**~~
- added "Italics" section
- moved "osu!" into its own section under "Grammar"
- moved a bunch of things
- re-read the entire thing and fixed a bunch of things
- added (better?) examples
- split sections into smaller, more meaningful(?) sections
- Fix grammar mistake in TOC
- remove dummy example in osu!
- small fix in Language Names
- small fix on Game Modes
- added another example for Gameplay Elements
- added another example for Contractions
- noted that possessive and contractions aren't the same
- noted that Section Headings should never have styles
- noted to not use "I" in Point of View
- added another example in Italics
- removed Clickable Images
- note flags
- reverted the "which site to link to" thing
- noted how to do **titles without translations**
- yay(?), section links
- new Lists section
- new Code Blocks
- noted reason why to use simplified chinese for zh files
- added new section "Language Tags"
- change note about osu!support inline-img-links to all images in general
- removed Capitalization section (no content)
  - lowered headings down of this section
- fixed `her's` to `hers` and `their's` to `theirs` (added `their`)
- list some table beautifying tools

### CODE REVIEW

**Comment by TPGPL** [comment]
new.ppy.sh is a temporary domain, so its wrong

File: wiki/Article_Style_Guide/en.md [hunk 1]
```diff
@@ -68,6 +79,22 @@ The title of an article is the name of the folder that the article is located in
...
+In the case where a title cannot be translated, use the language title (in English) followed by the language name (in that language) in round brackets.
+For example:
+
+```
+# osu!mania (Español)
+# Live Streaming (日本語)
+# Skinning osu!catch (Français)
+```
+
+You can find a list of language names in their language at [Language names in their own languages and scripts](http://www.omniglot.com/language/names.htm) or [List of language names](https://en.wikipedia.org/wiki/List_of_language_names).
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L97 : a bit redundant don't you think?

File: wiki/Article_Style_Guide/en.md [hunk 2]
```diff
@@ -181,67 +241,189 @@ For example, use:
...
+### Italics
+
+Any instances of the name of a videogame should also be in italics.
+For example:
+
+- `_Elite Beat Agents_`
+- `_Taiko no Tatsujin_`
 
-For example, instead of `Do NOT contact peppy for ingame issues!` use `Do **not** contact peppy for ingame issues!`
+As explain in the _osu!_ section, the word `osu!` (the game or framework itself) **must** be italicized.
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L277: put the osu!article ``As explained in the [osu!](wiki/article/here)...``

Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: Also there's a typo

File: wiki/Article_Style_Guide/en.md [hunk 3]
```diff
@@ -181,67 +241,189 @@ For example, use:
...
+
+```
+img/example.png
+./img/example.png
+../Insane/
+```
+
+Use of this type of link may include:
+
+- linking to an article that is a subdirectory of an article
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L397: end with period(?)

File: wiki/Article_Style_Guide/en.md [hunk 4]
```diff
@@ -181,67 +241,189 @@ For example, use:
...
+- linking to images for a specific article
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L398: end with period(?)

File: wiki/Article_Style_Guide/en.md (hunk 1 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: what about it is redundant?

File: wiki/Article_Style_Guide/en.md (hunk 2 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: what do you mean?! and where is the typo?

Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: ``As explan`` should be ``As explained``right?

:wink:

File: wiki/Article_Style_Guide/en.md (hunk 1 above)
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: ``Language name in their own languages [...]``

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: Rewording to use `list of native languages`

---

~~but wouldn't it be more confusing if it had said `You can find a list of language names at `... (I mean, in which language... in English?)~~

File: wiki/Article_Style_Guide/en.md [hunk 5]
```diff
@@ -116,11 +150,25 @@ Other examples may include:
...
+- `osu!standard`
+- `osu!direct`
+
+### Serial comma
+
+When listing items of 3 or more in a sentence, use the serial comma.
+
+- The game modes of _osu!_: osu!standard, osu!taiko, osu!catch**,** and osu!mania are fun to play with others.
+
+### Capitalisation
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L165: We wouldn't really need this afaik

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: ❓ the heading or the entire section (including the sub sections?)

Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: the whole **damn thing**

~~because literally, it's empty~~

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: why?

File: wiki/Article_Style_Guide/en.md [hunk 6]
```diff
@@ -68,6 +79,22 @@ The title of an article is the name of the folder that the article is located in
...
+```
+# Folder Name
+```
+
+In the case where a title cannot be translated, use the language title (in English) followed by the language name (in that language) in round brackets.
+For example:
+
+```
+# osu!mania (Español)
+# Live Streaming (日本語)
```
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: These examples just shows how to format the title if you **cannot** translate the title. (If you can translate the title, then just use the translation.)

~~Yes, most titles can be translated but a few (or more) articles were translated without it (or the transition from MediaWiki to Markdown got rid of them).~~

Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: yeah, most of the words like **osu!xxx**  can't be translate,but the others ~~just a few can't~~ :p

File: wiki/Article_Style_Guide/en.md [hunk 7]
```diff
@@ -68,6 +79,22 @@ The title of an article is the name of the folder that the article is located in
...
+
+```
+# Folder Name
+```
+
+In the case where a title cannot be translated, use the language title (in English) followed by the language name (in that language) in round brackets.
+For example:
+
+```
+# osu!mania (Español)
```
Comment (TPGPL) [inline wiki/Article_Style_Guide/en.md]: osu!mania is a bad example; you cant really translate it

File: wiki/Article_Style_Guide/en.md [hunk 8]
```diff
@@ -181,67 +241,248 @@ For example, use:
...
+- instead of `Do NOT contact peppy for ingame issues!` use `Do **not** contact peppy for ingame issues!`
+
+### Italics
+
+Any instances of the name of a videogame should also be in italics.
+For example:
+
+- `_Elite Beat Agents_`
+- `_Taiko no Tatsujin_`
+
```
Comment (TPGPL) [inline wiki/Article_Style_Guide/en.md]: Maybe another example that you can make text italic with "*"

File: wiki/Article_Style_Guide/en.md (hunk 7 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: that is the point, this section explains what to do if you can **not** translate a title

File: wiki/Article_Style_Guide/en.md [hunk 9]
```diff
@@ -101,26 +170,46 @@ For example use:
...
+When refering to `osu!` (the game/framework itself, not the game mode), it should be in _italics_ unless it is included in the name of game modes or other services related to the game.
+For example:
+
+- `The language that _osu!_ is written in is C# using the .NET Framework.`
+
+### Serial comma
+
+When listing items of 3 or more in a sentence, use the serial comma.
+
+- The game modes of _osu!_: osu!standard, osu!taiko, osu!catch**,** and osu!mania are fun to play with others.
```
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: what is that \*\* ?

File: wiki/Article_Style_Guide/en.md [hunk 10]
```diff
@@ -25,37 +25,77 @@ If you need help with GFM syntax, see [markdown-cheatsheet-online.pdf](https://g
...
+- `ja.md` for Japanese
+- `es.md` for Spanish
 
 Translated articles are to be placed in the appropriate English-named folder.
 
 For a list of the 2 lettered language codes, see [List of ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) (the 639-1 column of the table).
 
+## Language Tags
+
+Language tags are varients of a certain language (i.e. English is the language family, while American, British, Canadian, and Indian Enlish are varients).
```
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: e.g not i.e
you mentioned it in asg😂
~~maybe more~~

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: whoops, this is something I might not get used to :s

File: wiki/Article_Style_Guide/en.md (hunk 9 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: those `**,**` are trying to emphasize the comma (it is supposed to be the serial/oxford comma.

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: I'll just switch it over to code so it stands out more I guess

Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: i just mean those \*\* escape from markdown because of a missing space

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: ~~❓ I'm not really sure on what you mean about those `**`?~~ oh wait, are you saying that the missing space between `catch` and `**` is causing it to not be styled or something?

Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: ehh...actually, yes...

File: wiki/Article_Style_Guide/en.md [hunk 11]
```diff
@@ -181,67 +282,250 @@ For example, use:
...
 - `Do not` instead of `Don't`
 - `You will` instead of `You'll`
+- `It is` instead of `It's`
+
+Know that the possessive form of a word (e.g. `the player's` or `the skinner's`) are the expection (as these are not contractions).
+
+## Point of View
+
+When refering to the player, use `the player` or `a player`.
+**Never** use `he`/`his` or `she`/`her's`!
```
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: her's ???

File: wiki/Article_Style_Guide/en.md [hunk 12]
```diff
@@ -181,67 +282,250 @@ For example, use:
...
+Instead, use `they`/`their's`.
```
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: their's ???

File: wiki/Article_Style_Guide/en.md [hunk 13]
```diff
@@ -2,12 +2,12 @@
 
 The Article Style Guide serves as a way to help keep consistency in clarity, formatting, and layout between all articles of the osu!wiki.
 
-All articles should use plain English.
-Your word choice should explain such things in layman's terms (basically ask yourself, "If the reader is reading this, would they have to lookup any of the words?").
+All English articles should use plain English.
+Your word choice should help explain such things in layman's terms (basically ask yourself, "If the reader is reading this, would they have to lookup any of the words?").
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L6: Detail this a bit. Suggested ``... should be able to explain ...``

File: wiki/Article_Style_Guide/en.md [hunk 14]
```diff
@@ -65,34 +105,63 @@ Note that heading levels 3 and onwards use the hashtag style.
...
+
+```
+#### ![osu! icon](/wiki/shared/Osu.gif) osu!
+```
+
 ## TOC
 
 The TOC (Table of Contents) will automatically be generated on the osu!wiki.
-Because of this, there is no need to include add one.
+Because of this, there is no need to add one.
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L157-L160: Omit and Revise. Suggested ``ToCs are automatically generated in osu!web, and is no longer  a requirement``

File: wiki/Article_Style_Guide/en.md [hunk 15]
```diff
@@ -65,34 +105,63 @@ Note that heading levels 3 and onwards use the hashtag style.
...
 
 ## Grammar
 
-In the case where the American English and the British English spellings conflict, prefer the British Engish spelling.
+In the case where varients of English and the British English spellings conflict, prefer the British Engish spelling.
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L164: Suspected grammar error. Did you mean ``variants``?

File: wiki/Article_Style_Guide/en.md [hunk 16]
```diff
@@ -276,39 +559,71 @@ Names of images must be somewhat meaningful, please don't use the timestamp or l
...
 
 Tables use the GFM syntax.
 
-If you cannot create a table because _something_ won't work without HTML (i.e. lists inside tables), then you are overthinking both the table and the content.
+If you cannot create a table because _something_ won't work without HTML (e.g. lists inside tables), then you are overthinking both the table and the content.
 If you need help making a complicated table, try to see if you can improvise a simplier fix for it.
 
 **Never** place images inside tables.
 
+It is up to you whether or not if you want to beautify the tables.
```
Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: L602: Recommend beautification since there are more contributors that work on a similar article besides the reader.

File: wiki/Article_Style_Guide/en.md (hunk 14 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: would this do? `ToCs (Table of Contents) are automatically generated in the osu!web, because of this do not add one.`
... `no longer a requirement` sounds like it is optional

Comment (sr229) [inline wiki/Article_Style_Guide/en.md]: ``ToCs (Table of Contents) are automatically generated in osu!web, you do not need to manually make one for the article you intend to edit.`` Is my final suggestion

uwu

File: wiki/Article_Style_Guide/en.md (hunk 15 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: yes, ~~I always pronounced it as `var-e-unts` and not `var-e-ants` or something along those lines~~

File: wiki/Article_Style_Guide/en.md (hunk 14 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: sold, I'll take it

File: wiki/Article_Style_Guide/en.md (hunk 15 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: there were 2 other times this happened, they were fixed

File: wiki/Article_Style_Guide/en.md (hunk 16 above)
Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: I would recommend it as well, but those whom text wrap will see them looking like this
image
instead of
image

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: whoops, not the right example, but you get the gist of it, if it was beautified

File: wiki/Article_Style_Guide/en.md [hunk 17]
```diff
@@ -627,9 +627,9 @@ For example:
 
 ## Miscellaneous
 
-- All folders **must** contain a page of some kind, even if they are _index_ pages (pages that just link to other pages)
-- All sections **must** contain some content, even if they are redirect links to other pages
-  - `Also see [Glossary](/wiki/Glossary)`
+- All folders **must** contain a page of some kind, even if they are _index_ pages (pages that link to other pages).
+- All sections **must** contain some content, even those who contains subsections.
+  - `_Also see [Glossary](/wiki/Glossary)._`
```
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: why use _italic_?

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: idk, wikipedia has their `See also: ...` in italics

File: wiki/Article_Style_Guide/en.md [hunk 18]
```diff
@@ -181,67 +279,251 @@ For example, use:
...
+
+### Code Blocks
 
-When noting a keyboard key, use capital letters for single characters and [Camel case](https://en.wikipedia.org/wiki/Camel_case) for modifers.
+When using code blocks, use the ` ``` ` (triple grave mark) syntax.
 
-When naming a file extension, add a period (`.`) then the file extension in lower case letters.
+The osu!wiki site has a feature where you _could_ use four spaces to trigger the code blocks; however, this is **heavily** discouraged.
 
 ## Links
```
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: note: do not use long links
see ppy/osu-web#1060

Comment (TPGPL) [inline wiki/Article_Style_Guide/en.md]: ?? lol

Comment (TPGPL) [inline wiki/Article_Style_Guide/en.md]: it's because there is no design for mobile yet

Comment (MegaApplePi) [inline wiki/Article_Style_Guide/en.md]: it can't be the long link; I had mentioned in the dev chat that it was the wiki container as benpigchu had just said... but I will note that all links must have link text (though it kinda has been stated already)
